src/main/java/uk/gov/hmcts/reform/em/hrs/domain/HearingRecording.java

```diff
@@ -81,6 +81,9 @@ public class HearingRecording {
     @OneToMany(cascade = CascadeType.ALL, mappedBy = "hearingRecording")
     private Set<HearingRecordingSegment> segments;
 
+    @OneToMany(cascade = CascadeType.ALL, mappedBy = "hearingRecording")
+    private Set<HearingRecordingSharee> sharees;
```

Comment:
this will have performance issue on retrieval of records and even on delete, you can find other way to delete all

Reply:
have investigated behaviour on call with @mdayican using demo environment
looks ok as is bidirectional with lazy fetch type for the mapped many to one